content/2017/12/12/introducing-herman.rst

```diff
@@ -0,0 +1,389 @@
+public: yes
+tags: [Herman, CSS, Sass, 'Design Systems', Code, 'Style Guides']
+author: miriam
+image:
+  - src: 'blog/2017/herman-intro/header.jpg'
+headline:
+  - tagline: 'An agile approach to design systems'
+summary: |
+  **Design systems streamline development, communication, and consistency** --
+  but often rely on dedicated teams and extended budgets.
+  We wanted a tool to help us create and maintain
```

Comment:
Instead of "We" perhaps "OddBird..."

Reply:
Why make this less personal? Is it really not obvious on the OddBird blog who "we" is?